examples/hif2a/generate_star_map.py

```diff
@@ -0,0 +1,264 @@
+# Construct a star map for the fep-benchmark hif2a ligands
+
+from pathlib import Path
+import numpy as np
+from rdkit import Chem
+from rdkit.Chem import rdFMCS, Draw
+
+import matplotlib.pyplot as plt
+
+root = Path(__file__).parent.parent.parent
+
+# 0. Get force field
+from ff import Forcefield
+from ff.handlers.deserialize import deserialize_handlers
+
+path_to_ff = str(root.joinpath('ff/params/smirnoff_1_1_0_ccc.py'))
+with open(path_to_ff) as f:
+    ff_handlers = deserialize_handlers(f.read())
+forcefield = Forcefield(ff_handlers)
+
+# 1. Get ligands
+# TODO: possibly Git submodule to fep-benchmark inside datasets/, rather than copying fep-benchmark into datasets
```

Comment:
I have a love-hate (mostly hate) relationship with git submodule, but there may be times when its appropriate.